src/VisualStudio/Core/Def/TableDataSource/Suppression/VisualStudioDiagnosticListTableCommandHandler.cs

```diff
@@ -234,20 +234,9 @@ async System.Threading.Tasks.Task<Solution> ConfigureSeverityAsync(CancellationT
         }
     }
 
-    private DiagnosticData? TryGetSingleSelectedEntry()
+    private static DiagnosticData? TryGetSingleSelectedEntry()
     {
-        if (_tableControl?.SelectedEntries.Count() != 1)
-        {
-            return null;
-        }
-
-        if (!_tableControl.SelectedEntry.TryGetSnapshot(out var snapshot, out var index) ||
-            snapshot is not AbstractTableEntriesSnapshot<DiagnosticTableItem> roslynSnapshot)
```

Comment:
the 'or' always applies here.  so this method always returns null.

Reply:
Why not nuke this method then?

Reply:
just making safe cut off points.  tehre are more followups.